lib/pluck/pluck.test.js

```diff
@@ -0,0 +1,16 @@
+import pluck from '.'
+
+test('it doesn\'t extract values of an object by keys if key is not an array', () => {
+  try {
```

Comment:
I think you could use '.toTrhow()' matcher, instead of catching error
https://facebook.github.io/jest/docs/en/using-matchers.html

Reply:
Of course :)

Reply:
Sure, thanks